opensrp-app/src/main/java/org/smartregister/util/JsonFormUtils.java

```diff
@@ -231,9 +233,9 @@ private static void addMultiSelectListObservations(@NonNull Event event, @NonNul
                 String fieldCode = jsonObject.optString(OPENMRS_ENTITY_ID);
                 String parentCode = jsonObject.optString(OPENMRS_ENTITY_PARENT);
                 String value = jsonValObject.optString(OPENMRS_ENTITY_ID);
-                String humanReadableValues = jsonValObject.optString(AllConstants.TEXT);
+                String humanReadableValues = jsonValObject.optString(TEXT);
```

Comment:
Could we remove the formatting changes in this pr? i.e. converting `AllConstants.TEXT` to `TEXT`?

Reply:
Done